transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -346,32 +410,35 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng>(
         ring_signatures,
         pseudo_output_commitments,
         range_proof_bytes,
+        proof_of_opening,
     })
 }
 
-/// Concatenates [message || pseudo_output_commitments || range_proof].
+/// Computes a merlin digest of message, pseudo_output_commitments, range proof,
```

Comment:
Thanks. This looks much better.

Reply:
I'm now not sure if I should actually do this, since it's a breaking change and if we remove proof of opening this may not have to be a breaking change...

pondering it

Reply:
I guess that since we are doing the block version thing and it's a hard cut-over anyways we might as well keep this